lib/slack.ex

```diff
@@ -171,6 +171,20 @@ defmodule Slack do
   end
 
   @doc """
+  Sends `text` and `attachments` to `channel` for the given `slack` connection.
+  """
+  def send_attachments(text, attachments, channel, slack) do
```

Comment:
Maybe use a guard statement to ensure `attachments` is a list and raise otherwise?